src/ontodia/data/sparql/sparqlDataProvider.ts

```
@@ -555,7 +555,9 @@ export function executeSparqlConstruct(
                     resolve(triples);
                 }
             });
-        });
+        }).catch(error => {
+            console.log(error);
```

I don't think catching and eating an error here is a good idea.